doc/source/whatsnew/v0.23.1.txt

```diff
@@ -137,4 +137,4 @@ Bug Fixes
 **Other**
 
 - Tab completion on :class:`Index` in IPython no longer outputs deprecation warnings (:issue:`21125`)
-- Bug preventing pandas being used on Windows without C++ redistributable installed (:issue:`21106`)
```

Comment:
Can we undo this `diff` here?

Reply:
Every thing looks OK in my local and in github. I don't know how this happen. Do I need to revert the PR and push again?

Reply:
No need to, just run (relative to your `pandas` code directory):

~~~
git checkout origin/master doc/source/whatsnew/v0.23.1.txt
~~~

Commit this change and push it.

Reply:
I'm going to do undo in my last commit and push again

Reply:
I have deleted the commit and done a new one. I don't understand I didn't touch that line and it's there again.

Reply:
I have done your suggestion. I hope it is OK

Reply:
CI looks good to me 🙂